### PR TITLE
MAUI 862727 - Added accessibility for Dataform and Segmented control

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
**Description**
I have added an accessibility topic for Dataform and Segmented controls and added an accessibility description in Scheduler control.

### CODE REVIEW

File: MAUI/DataForm/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Accessibility in .NET MAUI DataForm control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/accessibility.md:4]: Learn here about Accessibility support with the Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.

File: MAUI/Segmented-Control/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Segmented Control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).
```
Comment (karthikraja-arumugam) [inline MAUI/Segmented-Control/accessibility.md:4]: Learn here about Accessibility support with the Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).

File: MAUI/Segmented-Control/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+platform: maui
+control: Segmented (SfSegmented) control
```
Comment (karthikraja-arumugam) [inline MAUI/Segmented-Control/accessibility.md:6]: Ensure is this correct format

File: MAUI/DataForm/accessibility.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply MAUI/DataForm/accessibility.md:4]: Changed.

File: MAUI/Segmented-Control/accessibility.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply MAUI/Segmented-Control/accessibility.md:4]: Changed.

File: MAUI/Segmented-Control/accessibility.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply MAUI/Segmented-Control/accessibility.md:6]: Changed.

File: MAUI/Scheduler/agenda-view.md [hunk 4]
```diff
@@ -306,4 +306,30 @@ this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
 {% endhighlight %}
 {% endtabs %}
 
-![appointment-text-appearence-customization-in-maui-scheduler](images/agenda-view/appointment-text-appearence-customization-in-maui-scheduler.png)
\ No newline at end of file
+![appointment-text-appearence-customization-in-maui-scheduler](images/agenda-view/appointment-text-appearence-customization-in-maui-scheduler.png)
+
+## Appointment time format customization
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/agenda-view.md:311]: Change the appointment time format

File: MAUI/Scheduler/agenda-view.md [hunk 5]
```diff
@@ -306,4 +306,30 @@ this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
...
+You can customize the appointment time format of scheduler by using the `AppointmentTimeFormat`property of [AgendaView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerAgendaView.html).
+
+{% tabs %}
+{% highlight XAML hl_lines="5" %}
+
+ <scheduler:SfScheduler x:Name="Scheduler" 
+                        View="Agenda">
+    <scheduler:SfScheduler.TimelineView>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/agenda-view.md:319]: use agendaview instead Timeline
 <scheduler:SfScheduler.TimelineView>
        <scheduler:SchedulerTimelineView

Comment (TamilarasanGunasekaran) [reply MAUI/Scheduler/agenda-view.md:319]: Changed.

File: MAUI/Segmented-Control/accessibility.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+<td>
+Right arrow<br/><br/></td><td>Moves to the next item on the segmented control's right side. It does not make any selection and only navigates to the next item. If the focus border is on the last item, pressing the RightArrow does nothing.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves to the previous item on the segmented control's left side. It does not make any selection and only navigates to the previous item. If the focus border is on the first item, pressing the LeftArrow does nothing.<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+Pressing the Enter key when segment item in focus will make selection.<br/><br/></td></tr>
```
Comment (M-A32) [inline MAUI/Segmented-Control/accessibility.md:31]: Pressing the Enter key when the segment item is in focus will make a selection.<br/><br/></td></tr>

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/DataForm/accessibility.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Accessibility in .NET MAUI DataForm control | Syncfusion
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply MAUI/DataForm/accessibility.md:3]: Changed.

File: MAUI/Segmented-Control/accessibility.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply MAUI/Segmented-Control/accessibility.md:31]: Changed.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**